Start of 1.18.2 Port

CaliberZenith opened the pull request:
## Changes
~~~~~~~~~~~
> Updated the build.gradle to use 1.18.2 Forge.
> Changed the mappings from Parchment to Official.
> Commented WorldGen code in the HLWorldGen and HyperLighting class.
> Commented out erroneous code in the LiquidBlockRendererMixin class.
> Updated the mods.toml file to reflect the update to 1.18.2.
> Commented out the WorldGen bus in the RegistryHandler class.
~~~~~~~~~~~
## Bugs
~~~~~~~~~~~
> The Hanging Fire has issues with the bounding box on the underside, causing a crash whenever a block is placed below it.
> The Hanging Fire and the Advanced Candle has an issue where the block itself breaks whenever a block update occurs to the side of it.
~~~~~~~~~~~

CaliberZenith committed:
Start of 1.18.2 Port

Changes
~~~~~~~~~~~
> Updated the build.gradle to use 1.18.2 Forge.
> Changed the mappings from Parchment to Official.
> Commented WorldGen code in the HLWorldGen and HyperLighting class.
> Commented out erroneous code in the LiquidBlockRendererMixin class.
> Updated the mods.toml file to reflect the update to 1.18.2.
> Commented out the WorldGen bus in the RegistryHandler class.

Bugs
~~~~~~~~~~~
> The Hanging Fire has issues with the bounding box on the underside, causing a crash whenever a block is placed below it.
> The Hanging Fire and the Advanced Candle has an issue where the block itself breaks whenever a block update occurs to the side of it.

hypherionmc reviewed (changes requested):
Please update TOP and Cloth Config as well

hypherionmc reviewed (changes requested):
Don't see your name in the "Authors" section

CaliberZenith committed:
Update gradle.properties

Changes
~~~~~~~~~~
> Updated Cloth Config from version 6.1.48 to 6.2.57 in gradle.properties.
~~~~~~~~~~

hypherionmc committed:
Small Code Cleanup

hypherionmc committed:
Updated Hypherion's Maven & remove RGBLib

hypherionmc committed:
Added Misha as an Author / Cleanup

hypherionmc committed:
Fixed HangingFire game crash when placing block below it

hypherionmc committed:
Re-add World Gen for Colored Water
